core/msd/src/main/rdl/TransportPolicyRule.tdl

```diff
@@ -15,6 +15,19 @@ type TransportPolicyProtocol Enum {
     UDP
 }
 
+// Validation Status of transport policy vs network policy
+type TransportPolicyValidationStatus Enum {
+   ALL,
```

Comment:
can we make this explicit? Current values are a little ambiguous, e.g. its not clear whether ALL means ALL policies are good or ALL are bad.